### PR TITLE
BUGFIX: SD-497 Add combo2 to supported stacking chart type

### PR DESCRIPTION
**Pull request by nadtan**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-app-components:
- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by nadtan** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2119/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1469/ : UNSTABLE in 13m 28s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4912/ : SUCCESS in 13s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3879/ : SUCCESS in 22m 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4911/ : SUCCESS in 10s

**Comment by nadtan** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3891/ : SUCCESS in 6m 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4929/ : SUCCESS in 7s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2125/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1477/ : SUCCESS in 10m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4930/ : SUCCESS in 8s

**Comment by nadtan** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1020.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1478/ : SUCCESS in 24m 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4931/ : SUCCESS in 8s

File: stories/test_data/fixtures.ts [hunk 1]
```diff
@@ -288,6 +288,14 @@ export const comboWithTwoMeasuresAndViewByAttributeMdObject: VisualizationObject
         .buckets,
 };
 
+export const comboWithThreeMeasuresAndViewByAttribute = barChartWith3MetricsAndViewByAttribute;
+export const comboWithThreeMeasuresAndViewByAttributeMdObject: VisualizationObject.IVisualizationObjectContent = {
+    buckets: require("../test_data/combo_chart_with_three_measures_view_by_attribute_md_object.json").buckets,
+    filters: require("../test_data/combo_chart_with_three_measures_view_by_attribute_md_object.json").buckets,
```
Comment (ivan-nejezchleb) [inline stories/test_data/fixtures.ts]: 'filters' and 'visualizationClass' from this are not used in code (logic is driven only by buckets), but still I would not put there 'buckets' from json file

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3932/ : SUCCESS in 24m 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4979/ : SUCCESS in 12s

File: stories/test_data/fixtures.ts (hunk 1 above)
Comment (nadtan) [reply stories/test_data/fixtures.ts]: 'filters' and 'visualizationClass' are removed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3951/ : SUCCESS in 9m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5005/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3953/ : SUCCESS in 17m 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5008/ : SUCCESS in 11s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2158/ : SUCCESS in 10m 01s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3954/ : SUCCESS in 4m 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1041/ : SUCCESS in 24s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1204/ : SUCCESS in 10m 17s

**Pull request merged**